### ISSUE TITLE
partial class modifier wrong in FakeQueryProvider<T>

### ISSUE THREAD

**Issue by FaizulHussain** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#692]
When using the "public partial" class modifier, it's generating the FakeQueryProvider<T> as:

_public **partial abstract** class FakeQueryProvider<T> : IOrderedQueryable<T>, IQueryProvider_

The abstract keyword should precede partial

Thanks

**Comment by sjh37** [comment]
Thanks @FaizulHussain

**Comment by PatrikSoder** [comment]
Hello, we have run into the same issue, is there any way to fix it in the tt-file or include file ourselves ?

**Comment by sjh37** [comment]
Appoligies for the error.
In `EF.Reverse.POCO.v3.ttinclude` please change both instances of:
```cs
{{DbContextClassModifiers}} abstract class FakeQueryProvider<T> : IOrderedQueryable<T>, IQueryProvider{{#newline}}
```

to
```cs
public abstract class FakeQueryProvider<T> : IOrderedQueryable<T>, IQueryProvider{{#newline}}
```

**Comment by PatrikSoder** [comment]
thanks for the super fast reply @sjh37 !

**Comment by sjh37** [comment]
Now released in v3.5.0
